### PR TITLE
Feature/uhubctl power control

### PR DESCRIPTION
**Pull request by Pixep**
Add power control via uhubctl utility for USB hubs

### CODE REVIEW

File: pluma/core/power/uhubctl.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+class Uhubctl(PowerBase):
+    ''' USB hub power control using uhubctl utility '''
+
+    def __init__(self, location: str, port: int, reboot_delay: int = None):
+        PowerBase.__init__(self, reboot_delay)
+        self.location = location
+        self.port = port
+
+        if not location or not port:
+            raise Exception('Uhubctl: "location" and/or "port" '
```
Comment (mjftw) [inline pluma/core/power/uhubctl.py:17]: Would be better to raise a PDUError here.
https://github.com/Witekio/pluma-automation/blob/3265febdf77a33cc74f92eb888138040cc24b4dc/pluma/core/pdu.py#L11

Now that I look at it, maybe these should be called `PowerError` instead, nobody knows that `PDU` stands for Power Distribution Unit. Something to be put on another issue maybe.

**Review by mjftw — APPROVED** [review]
I've never used uhubctl before, looks pretty cool!

Since we're adding a new power controller, it would be good to add some reference to this in the docs (maybe in supported hardware).

**Review by mjftw — CHANGES_REQUESTED** [review]
Sorry meant to stick it as request changes for the Exception thing

File: pluma/core/power/uhubctl.py (hunk 1 above)
Comment (mjftw) [reply pluma/core/power/uhubctl.py:17]: https://github.com/Witekio/pluma-automation/blob/master/docs/style-guide.md#exceptions

Comment (Pixep) [reply pluma/core/power/uhubctl.py:17]: Good point. Changed to `PDUError`

**Comment by Pixep** [comment]
> Since we're adding a new power controller, it would be good to add some reference to this in the docs (maybe in supported hardware).

Added to the list!

**Review by mjftw — APPROVED** [review]

**Pull request merged**